infracost-usage-example.yml

```diff
@@ -14,6 +14,16 @@ resource_usage:
   # module.lambda_function.aws_lambda_function.this[0]:
   #   monthly_requests: 20000
   #   request_duration_ms: 600
+  #
+  # You can use [*] for array of resources to apply usage-values individually to each element of the array.
+  # Specifying an array element have priority.
```

Comment:
If this what we mean here?
```
If usage for an array element (e.g. this[0]) and [*] exist, only the array element's usage will be applied to that resource.
```

Reply:
Yes it is. Should I fix it to your describe?

Reply:
Thanks, that would be good.